src/storage/backend.rs

```diff
@@ -0,0 +1,197 @@
+//! StorageBackend that uses a local filesystem, like a traditional FTP server.
```

Comment:
I would rather name this file `storage.rs` or `storage_backend.rs` `backend.rs` is too general

Reply:
will do

Reply:
thinking was that if you import it you say: `use storage::backend`, hmmm

Reply:
I tried to rename to storage and clippy doesn't like it:

```
    Checking libunftp v0.7.0 (/Users/hdejager/Develop/unftp/libunftp)
error: module has the same name as its containing module
 --> src/storage/mod.rs:8:1
  |
8 | pub(crate) mod storage;
  | ^^^^^^^^^^^^^^^^^^^^^^^
 
```

Reply:
named it storage_backend